packages/devextreme/js/core/utils/license.js

```diff
@@ -0,0 +1,43 @@
+import { parseToken } from '../../__internal/core/license/license_validation';
+import errors from '../errors';
+
+let isLicenseVerified = false;
+
+export function verifyLicense(licenseToken, version) {
+    if(isLicenseVerified) {
+        return;
+    }
+    isLicenseVerified = true;
+
+    let warning;
+
+    if(licenseToken) {
+        const license = parseToken(licenseToken);
+
+        if(license.kind === 'corrupted') {
```

Comment:
Comment for future TS :)

Let's move the type of kind to a separate const in ```license_validation.ts```
And use this const here